static/main.scss

```diff
@@ -270,6 +271,18 @@ body, pre {
         @extend .octicon-git-branch;
         margin-right: 5px;
       }
+
+      &.clean {
+        color: #8AD843;
+      }
+
+      &.modified {
+        color: #E9C059;
+      }
+
+      &.deleted {
+        color: #D75857;
+      }
```

Comment:
There is a file `colors.scss`. We have to fetch colors from it, because the actual colors will change with themes.